core/tauri/src/manager.rs

```diff
@@ -504,9 +504,9 @@ impl<R: Runtime> WindowManager<R> {
         let parsed_path = Url::parse(request.uri())?;
         let filtered_path = &parsed_path[..Position::AfterPath];
         #[cfg(target_os = "windows")]
-        let path = filtered_path.replace("asset://localhost/", "");
+        let path = filtered_path.trim_start_matches("asset://localhost/");
```

Comment:
`trim_start_matches` will remove the match multiple times; is that really desirable?
I think [`strip_prefix`](https://doc.rust-lang.org/std/primitive.str.html#method.strip_prefix) is a little more appropriate here.

Is there any reason why the path wouldn't start with that prefix at this point?

Reply:
You're right lol